### DISCUSSION TITLE
Post: Use Encrypted DNS

### DISCUSSION THREAD

**Discussion by giscus** [rkt2spc/blog#1] [General]
# Post: Use Encrypted DNS

A guide on how to use encrypted DNS

https://rkt2spc.com/blog/use-encrypted-dns

**Comment by kronos** [comment]
Привет, комрад

**Reply by rkt2spc** [reply]
Спасибо, комрад!

**Comment by htranugaming** [comment]
Cưng lắm Tuấn :D
